### DISCUSSION TITLE
Transferred database to hobby account but cannot locate it

### DISCUSSION THREAD

**Discussion by joshuaeilers** [vercel/community#4923] [Account Issue] [answered]
### Question

Hello,

I was working through the next.js learning tutorial, created a db called `nextjs-dashboard-postgres`. I realized I accidentally created it under my work team, so I transferred it to my hobby account. But I couldn't locate it under my hobby account, so I made a new one with the same name. The old one is still deployed out there somewhere, and I have the env vars (url/password/etc) to access it. I'd like to remove the old one somehow, if anyone knows of a way I would appreciate any tips if I'm missing something.

**Comment by amyegan** [comment, marked as answer]
Hi Joshua! I found the duplicate database and we can remove it for you. I just want to double check that you have everything you need on the new one before we delete the old one (endpoint `ep-patient-hall...`). Once you confirm, I can get the original db removed for you

**Reply by joshuaeilers** [reply]
Yeah, the ep-patient-hall is the right one to remove. Just the seed data from the nextjs learning course. Please remove when is convenient, thank you!

**Reply by amyegan** [reply]
Done! 😄
